src/obr/core/queries.py

```diff
@@ -185,13 +231,22 @@ def query_to_dict(
 
 
 def query_impl(
-    jobs: OpenFOAMProject, queries: list[Query], output=False, latest_only=True
+    jobs: "Union[OpenFOAMProject, list[Job]]",
+    queries: list[Query],
+    output=False,
+    latest_only=True,
 ) -> list[str]:
     """Performs a query and returns corresponding job.ids"""
     res = query_to_dict(jobs, queries, output, latest_only)
     if output:
+        if len(queries) > 0:
+            q = queries[0]
+            if len(res) == 0:
+                logging.info(f"No results for Query {q}")
+            else:
+                logging.info(f"Query results for {q}:")
         for r in res:
-            logging.info(r)
+            logging.info(f"\t{r}")
```

Comment:
This prints out all results even if they are empty. I guess we could either fix this in this PR or since I think we should also rework the queries cli API we could do this in a separate PR

Reply:
I would prefer to do it in a separate PR. I will create an issue for it later.